mat44xmat44.c

```diff
@@ -0,0 +1,115 @@
+#include <immintrin.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/time.h>
+
+#define N 4
+
+int
+main() {
+
+    // allocate memory for array
+    int are_equal = 1;
+    int count = 0;
+    float(*A)[N];
```

Comment:
some comments, because the arrays are so small (4x4), you don't need to do dynamic allocation or alignment with posix_memalign. you could just define float A[4][4].
And for alignment you could just add __attribute__((aligned(16)) in there.